example_configs/dropwizard-0-8-x.yml

```diff
@@ -0,0 +1,153 @@
+  name: log_msg_all_rate
+  labels:
+    calculation: "$1"
+  type: COUNTER
+
+- pattern: "metrics<name=org.eclipse.jetty.server.HttpConnectionFactory.(.+).connections><>Count"
+  name: http_connections_total
+  type: COUNTER
+  labels:
+    port: "$1"
+
+
+- pattern: "metrics<name=io.dropwizard.jetty.MutableServletContextHandler.(.+)-responses><>Count"
+  name: http_response_codes_total
+  type: COUNTER
+  labels:
+    http_code: "$1"
+
+- pattern: "metrics<name=io.dropwizard.jetty.MutableServletContextHandler.active-(.+)><>Count"
+  name: http_active_$1
+  type: GAUGE
+
+
+- pattern: "metrics<name=io.dropwizard.jetty.MutableServletContextHandler.async-(.+)><>Count"
+  name: http_request_async_$1_total
+  type: COUNTER
+
+- pattern: "metrics<name=io.dropwizard.jetty.MutableServletContextHandler.async-(.+)><>(FiveMinuteRate|MeanRate|OneMinuteRate|FifteenMinuteRate)"
+  name: http_request_async_$1_rate_seconds
+  type: GAUGE
+  labels:
+    calculation: "$2"
+
+
+- pattern: "metrics<name=io.dropwizard.jetty.MutableServletContextHandler.(.+)-requests><>Count"
+  name: http_request_total
+  type: COUNTER
+  labels:
+    http_method: "$1"
+
+- pattern: "metrics<name=io.dropwizard.jetty.MutableServletContextHandler.(.+)-requests><>(FiveMinuteRate|MeanRate|OneMinuteRate|FifteenMinuteRate)"
+  name: http_request_rate
+  type: GAUGE
+  labels:
+    http_method: "$1"
+    calculation: "$2"
+
+- pattern: "metrics<name=io.dropwizard.jetty.MutableServletContextHandler.(.+)-requests><>(Max|MinMean|StdDev|50thPercentile|75thPercentile|95thPercentile|98thPercentile|99thPercentile|999thPercentile)"
+  name: http_request_duration_milliseconds
+  type: GAUGE
+  labels:
+    http_method: "$1"
+    calculation: "$2"
+
+
+- pattern: "metrics<name=io.dropwizard.jetty.MutableServletContextHandler.requests><>Count"
+  name: http_all_request_total
+  type: COUNTER
+
+- pattern: "metrics<name=io.dropwizard.jetty.MutableServletContextHandler.requests><>(FiveMinuteRate|MeanRate|OneMinuteRate|FifteenMinuteRate)"
+  name: http_request_all_rate_seconds
+  type: GAUGE
+  labels:
+    calculation: "$1"
+
+
+- pattern: "metrics<name=io.dropwizard.jetty.MutableServletContextHandler.requests><>(Max|MinMean|StdDev|50thPercentile|75thPercentile|95thPercentile|98thPercentile|99thPercentile|999thPercentile)"
+  name: http_all_request_duration_milliseconds
+  type: GAUGE
+  labels:
+    calculation: "$1"
+
+
+- pattern: metrics<name=io.dropwizard.jetty.MutableServletContextHandler.dispatches><>(FiveMinuteRate|MeanRate|OneMinuteRate|FifteenMinuteRate)
+  name: http_dispatches_rate_seconds
+  type: GAUGE
+  labels:
+    calculation: "$1"
+
+
+- pattern: metrics<name=io.dropwizard.jetty.MutableServletContextHandler.dispatches><>(Max|MinMean|StdDev|50thPercentile|75thPercentile|95thPercentile|98thPercentile|99thPercentile|999thPercentile)
+  name: http_dispatches_duration_milliseconds
+  type: GAUGE
+  labels:
+    calculation: "$1"
+
+
+- pattern: "^metrics<name=(jvm.+)><>Count"
```

Comment:
You should drop all these, we already expose them automatically and in a more prometheus-friendly way